pgdrive/pg_config/collision_group.py

```diff
@@ -0,0 +1,38 @@
+from panda3d.bullet import BulletWorld
+
+
+class CollisionGroup:
+    Terrain = 2
+    Ego_Vehicle = 1
+    Lane_line = 3
```

Comment:
Would you mind rename this to "Lane_Line" or even "LaneLine" (Big camel) as well as other variable in here.